.github/workflows/server-ci-artifacts.yml

```diff
@@ -89,22 +67,24 @@ jobs:
           echo "| --- |" >> "${GITHUB_STEP_SUMMARY}"
           for package in ${PACKAGES_FILE_LIST}
             do 
-              echo "|[${package}](https://s3.amazonaws.com/pr-builds.mattermost.com/mattermost/commit/${{ github.event.workflow_run.head_sha }}/${package})|" >> "${GITHUB_STEP_SUMMARY}"
+              echo "|[${package}](https://pr-builds.mattermost.com/mattermost/commit/${{ github.event.workflow_run.head_sha }}/${package})|" >> "${GITHUB_STEP_SUMMARY}"
           done
-        if: needs['determine-if-monorepo'].outputs.is_monorepo == 'true'
 
   build-docker:
     runs-on: ubuntu-22.04
     needs:
       - upload-artifacts
-      - determine-if-monorepo
     steps:
       - name: cd/docker-login
         uses: docker/login-action@3da7dc6e2b31f99ef2cb9fb4c50fb0971e0d0139 # v2.1.0
         with:
           username: mattermostdev
           password: ${{ secrets.DOCKERHUB_DEV_TOKEN }}
-        if: needs['determine-if-monorepo'].outputs.is_monorepo == 'true'
+
+      - name: cd/setup-cosign
+        uses: sigstore/cosign-installer@11086d25041f77fe8fe7b9ea4e48e3b9192b8f19 # v3.1.2
+        with:
+          cosign-release: v${{ env.COSIGN_VERSION }}
```

Comment:
I may have missed but I didn't see where cosign is applied/used. Is this preparatory setup only? Btw, did we have a prior approval/agreement from SecOps to proceed in using cosign?

Reply:
I am just using it to get the digest from Dockerhub . I am not signing anything 😄